index.bs

```diff
@@ -7155,6 +7155,45 @@ partial dictionary MLOpSupportLimits {
     ::
         The interpolation algorithm used to fill the output tensor values.
 
+        Both algorithms start with these inputs, computed for each spatial axis:
+        ```
+        scale = outputSize / inputSize
+        unclampedCoordinate = (outputCoordinate + 0.5) / scale - 0.5
+        inputCoordinate = max(min(unclampedCoordinate, inputSize - 1), 0)
+        ```
+        For a given `outputCoordinate.x` and `outputCoordinate.y` location in the output tensor, the above equations give a rational `inputCoordinate.x` and `inputCoordinate.y`.
+
+        <dl dfn-type=enum-value dfn-for=MLInterpolationMode>
+        : <dfn>nearest-neighbor</dfn>
+        ::
+            The `inputCoordinate.x` and `inputCoordinate.y` computed above are used as inputs to a nearest-neighbor sampling algorithm to compute the output tensor value as follows:
+            ```
+            x = ceil(inputCoordinate.x - 0.5)
+            y = ceil(inputCoordinate.y - 0.5)
+            output tensor value = input tensor value at (x, y)
+            ```
+
+        : <dfn>linear</dfn>
+        ::
+            The `inputCoordinate.x` and `inputCoordinate.y` computed above are used as inputs to a bilinear sampling algorithm to compute the output tensor value as follows:
+            ```
+            x0 = floor(inputCoordinate.x)
+            x1 = ceil(inputCoordinate.x)
+            y0 = floor(inputCoordinate.y)
+            y1 = ceil(inputCoordinate.y)
+            v00 = input tensor value at (x0, y0)
+            v10 = input tensor value at (x1, y0)
+            v01 = input tensor value at (x0, y1)
+            v11 = input tensor value at (x1, y1)
+            tx = inputCoordinate.x - floor(inputCoordinate.x)
+            ty = inputCoordinate.y - floor(inputCoordinate.y)
+
+            a = v00 * (1 - tx) + v10 * tx;
+            b = v01 * (1 - tx) + v11 * tx;
+            output tensor value = a * (1 - ty) + b * ty
```

Comment:
```suggestion
            vy0 = vx0y0 * (1 - tx) + vx1y0 * tx;
            vy1 = vx0y1 * (1 - tx) + vx1y1 * tx;
            output tensor value = vy0 * (1 - ty) + vy1 * ty
```
Just a little clearer and consistent naming of the y interpolants with the x interpolants.